txs/names.go

```diff
@@ -31,22 +27,8 @@ var (
 
 	MaxNameLength = 64
 	MaxDataLength = 1 << 16
```

Comment:
Curious what this means?

Reply:
i thought i remembered it was 2^16. was just weirded out by the go syntax. that's a lot of bytes....